policyengine_canada/variables/gov/provinces/yt/tax/income/credits/yt_basic_personal_amount.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_canada.model_api import *
+
+
+class yt_basic_personal_amount(Variable):
+    value_type = float
+    entity = Person
+    label = "Yukon basic personal amount"
+    definition_period = YEAR
+    defined_for = ProvinceCode.YT
+
+    def formula(person, period, parameters):
+        income = person("individual_net_income", period)
+        p = parameters(
+            period
+        ).gov.provinces.yt.tax.income.credits.basic_personal_amount
+        exceedance = income - p.income_threshold
```

Comment:
if the person has income of under $165,430 they just get the main $13,521 - we need to account for that